sdk/servicebus/service-bus/src/receivers/receiver.ts

```diff
@@ -323,7 +323,7 @@ export class ReceiverImpl<ReceivedMessageT extends ReceivedMessage | ReceivedMes
       connectionId: this._context.namespace.connectionId,
       operation: receiveMessages,
       operationType: RetryOperationType.receiveMessage,
-      abortSignal: undefined,
+      abortSignal: options?.abortSignal,
```

Comment:
The only things that are left out in this PR are streaming receiver and the send operations.

Reply:
send operations & streaming receiver do not respect abort signal at the moment, @richardpark-msft is working on it